web_static/usage.html

```diff
@@ -370,7 +370,5 @@ <h2><a name="how">How It Works (and Why It's So Slow Sometimes)</a></h2>
 using the service.</p>
 
 <hr>
-<address>updated March 24, 2009 - egnor<script>
-document.write("@");
-</script>ofb.net</address>
+<address>updated March 24, 2009 - egnor<script>document.write("@");</script>ofb.net</address>
```

Comment:
You can just put in my real email, no need for any obfuscation, I'm long past trying to hide it from the spammers lol